### PR TITLE
Mvh admin merch dashboard 39

### PR DESCRIPTION
**Pull request by mikez321**
Pulling US39 into US38.  Adds tests and functionality for admin disabling merchant accounts.  Adds refreshes to tests so they all pass.

### CODE REVIEW

File: app/views/admin/merchants/index.html.erb [hunk 1]
```diff
@@ -1,14 +1,14 @@
 <h1>All Merchants</h1>
 <section>
-  <% @merchants.each do |merchant| %>
+  <% @merchants.map do |merchant| %>
```
Comment (mikez321) [inline app/views/admin/merchants/index.html.erb:3]: Probably should change this back to an each...

File: app/views/admin/merchants/index.html.erb [hunk 2]
```diff
@@ -8,7 +8,7 @@
       <% if merchant.status == "enabled" %>
         <%= button_to "Disable", "/admin/merchants/#{merchant.id}", method: :patch %>
       <% end %>
-  </ul>
-  <% end %>
+    </ul>
   </div>
+  <% end %>
```
Comment (mikez321) [inline app/views/admin/merchants/index.html.erb:13]: Good catch here Kevin!

File: app/views/admin/merchants/index.html.erb (hunk 1 above)
Comment (kmcgrevey) [reply app/views/admin/merchants/index.html.erb]: I agree -- this is cute Ruby -- but if we need more links and info added to each `merchant` this makes that difficult

File: spec/features/admin/admin_merch_dashboard_spec.rb [hunk 3]
```diff
@@ -72,25 +75,48 @@
...
+    visit "/admin/merchants"
+
+    within ".merchant-#{@meg.id}" do
+      click_button("Disable")
+    end
+    @meg.reload
+    @tire.reload
+    @seat.reload
+    @pump.reload
+    expect(@meg.status).to eq ("disabled")
```
Comment (kmcgrevey) [inline spec/features/admin/admin_merch_dashboard_spec.rb:116]: mystery solved!  we so needed this `reload` . . . now we can trust our testing again!

**Comment by kmcgrevey** [comment]
Looks good.  Let's keep an eye on that `map do` in the admin-merchant-index page.

**Pull request merged**